examples/e-commerce/src/widgets/Configuration.js

```diff
@@ -3,6 +3,7 @@ import { configure } from 'instantsearch.js/es/widgets';
 const configuration = configure({
   attributesToSnippet: ['description:10'],
   snippetEllipsisText: '…',
+  removeWordsIfNoResults: 'allOptional',
```

Comment:
I don't understand `allOptional` it would have made more sense to me if you've set it to `none`.
This makes it harder to actually see the "no results/empty" state right ?

Reply:
Yes, which is the goal for an e-commerce use case usually. You don't want to show that there's no results often.